background.ts

```diff
@@ -99,58 +95,57 @@ app.whenReady().then(async() => {
   window.openPreferences();
 });
 
-app.on('before-quit', (event) => {
+Electron.app.on('before-quit', async(event) => {
```

Comment:
TypeScript already knows that this is an `Electron.Event` because they generate TypeScript declarations [via docs](https://github.com/electron/typescript-definitions).

Reply:
good to know